src/google/cloud/ndb/model.py

```diff
@@ -91,8 +92,6 @@
     "MetaModel",
     "Model",
     "Expando",
-    "transaction",
```

Comment:
These were already implemented in _transaction.py. Belatedly removing their stubs from model.py